common/tier4_camera_view_rviz_plugin/src/bird_eye_view_controller.cpp

```diff
@@ -0,0 +1,242 @@
+// Copyright 2023-2024 Autoware Foundation
```

Comment:
When publishing a work created in 2023, it is common practice to list only that year. It is not usual to list future years in advance.
Do you have any references for future copyrights?

Reply:
Thanks for pointing out this fault. It was some naive personal thoughts. I checked some references and I would align to your suggestion.